pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/bestpractices/WhileLoopWithLiteralBoolean.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.bestpractices;
+
+import net.sourceforge.pmd.testframework.PmdRuleTst;
+
+public class WhileLoopWithLiteralBoolean extends PmdRuleTst {
```

Comment:
This class needs to be suffixed with "Test" - otherwise the test is not picked up by surefire...